modules/social_features/social_group/modules/social_group_default_route/social_group_default_route.module

```diff
@@ -0,0 +1,158 @@
+<?php
+
+/**
+ * @file
+ * Default route for groups.
+ */
+
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\group\Entity\GroupType;
+
+/**
+ * Implements hook_form_alter().
+ */
+function social_group_default_route_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+
+  $forms = [];
+  foreach (_social_group_default_route_group_types() as $type) {
```

Comment:
@jochemvn Is it safe to use constructions like this? What If this function will return an empty array or not array?

Reply:
👌  I've added an social_group_default.api.php file to document the hook